lib/transformer/caster/exporter.rb

```diff
@@ -0,0 +1,33 @@
+module Transformer
+  module Caster
+    class Exporter < Base
+      GraphMethodValue = Struct.new(:method, :key, :value)
+
+      def analyze
+        exportable_attributes.map do |key, opts|
+          @template.add_graph_value(opts.key, opts.method, opts.value)
+        end
+
+        @template
+      end
+
+      private
+
+      def exportable_attributes
+        result = {}
+
+        @dataset_cast.graph_methods.each_pair do |key, value|
+          opts = GraphMethods.all[key]
+
+          if opts && value
```

Comment:
Regarding `if opts` ... Is it *possible* for `@dataset_cast.graph_methods` to include values for which there is no corresponding `GraphMethod`? Could this hide errors? (for example, if a Caster assigns a value for a graph method which has been removed)

Reply:
> @dataset_cast.graph_methods

They shouldn't. It should be added to the validations 👍